Interni/Verbali Interni/2020_11_09/components/verbale.tex

```diff
@@ -0,0 +1,96 @@
+\section{Infomazioni generali}
+
+\begin{itemize}
+
+	\item \textbf{Canale di comunicazione:} Zoom;
+	
+	\item \textbf{Data:} \DataMeeting{};
+	
+	\item \textbf{Ora inizio:} 21;
+	
+	\item \textbf{Ora fine:} 22;
+	
+	\item \textbf{Segretario:} Francesco De Marchi;
+	
+	\item \textbf{Partecipanti:}
+	
+		\begin{itemize}
+		
+			\item Daniele Giachetto;
+			\item Davide Albiero;
+			\item Francesco De Marchi;
+			\item Giosuè Calgaro;
+			\item Lucrezia Gradi;
+			\item Matteo Pagotto;
+			\item Tommaso Poppi.
+				 
+		\end{itemize}
+
+\end{itemize}
+
+\section{Ordine del giorno}
+
+\begin{itemize}
+
+	\item\textbf{ Presentazione membri del gruppo;}
+
+	\item\textbf{ Votazione su canali di comunicazione da utilizzare;}
+
+	\item\textbf{ Impostazione dei canali di comunicazione;}
+
+	\item\textbf{ Votazione sulla registrazione delle chiamate;}
+
+	\item\textbf{ Confronto sugli orari e sulla frequenza delle riunioni.}
+
+
+\end{itemize}
+\newpage
+
+\section{Resoconto}
+
+\subsection{Presentazione membri del gruppo}
+
+I membri del gruppo si sono presentati, permettendo quindi di capire gli esami arretrati, le esperienze lavorative e scolastiche ed inoltre gli impegni di ogni membro.
+
+\subsection{Votazione su canali di comunicazione da utilizzare}
+
+Dopo una discussione nella quale sono stati confrontati diversi software, sono stati scelti Slack per la comunicazione testuale e Zoom per la comunicazione audio e video
+
+\subsection{Impostazione dei canali di comunicazione}
+
+È stato creato un canale Slack ed è stato opportunamente impostato in vari sottocanali per permettere una migliore organizzazione ed una più chiara comunicazione tra membri
+
+\subsection{Votazione sulla registrazione delle chiamate}
+
+Sotto proposta di Daniele è stato deciso all’unanimità di registrare le chiamate, questo perchè la registrazione ed il salvataggio delle chiamate risultano un peso trascurabile mentre i benifici che esse possono portare sono diversi.
```

Comment:
Scritto male benefici

Reply:
Il simbolo dell'apostrofo a Latex non gli va bene🤷‍♂️

Reply:
Sottocanali non va attaccato, staccato o con la linea in mezzo

Reply:
Fixato 1 e 3, per il 2 ho cambiato il simbolo dell'apostrofo, spero non lo riconverta automaticamente